pkg/deploy/devconfig.go

```diff
@@ -189,6 +198,7 @@ func DevConfig(_env env.Core) (*Config, error) {
 				"EnableOCMEndpoints",
 				"RequireOIDCStorageWebEndpoint",
 				"UseMockMsiRp",
+				"EnableMISE",
```

Comment:
Do we actually want this feature on for full-service RPs (where this devconfig gets used)? 

If it works fine in that context, I say keep it in, but if it doesn't work at all or causes other issues we may want to disable this for full-service.

Reply:
For full-service RP there is no involvement of ARM so MISE will not be needed here. However, we have added test detail for the config to pass validation. Ideally this should work, but I am yet to create a full-service RP setup to test this practically.

Reply:
@tsatam While creating this, the VMs had MISE and OTEL coming up as expected but rp.portal were failing due other unrelated issues. I have removed the feature flag and added instruction for mirroring MISE image from int for letting full-rp-service setup work as it is for now.